db/migration/1729600015045-AddMultiDimIdConfigMd5SlugIndex.ts

```diff
@@ -0,0 +1,30 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class AddMultiDimIdConfigMd5SlugIndex1729600015045
+    implements MigrationInterface
+{
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`-- sql
+            ALTER TABLE multi_dim_data_pages
+            DROP PRIMARY KEY,
+            ADD COLUMN id SERIAL PRIMARY KEY FIRST,
```

Comment:
I looked up `SERIAL` because I wasn't familiar with it, and it turns out it is a bit funky: https://stackoverflow.com/a/65926920

It's possibly better to use this (same in the other migration):

```suggestion
            ADD COLUMN id INT unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY FIRST,
```

Reply:
Good catch, I should have checked more thoroughly when Copilot generated it. Changed.